src/main/java/redis/clients/jedis/BinaryJedis.java

```diff
@@ -2865,7 +2886,7 @@ public Long zcount(final byte[] key, final byte[] min, final byte[] max) {
 
   protected Set<Tuple> getTupledSet() {
     List<byte[]> membersWithScores = client.getBinaryMultiBulkReply();
-    if (membersWithScores.isEmpty()) {
+    if (membersWithScores == null || membersWithScores.isEmpty()) {
```

Comment:
If response from Redis `nil`, we should return `null`.

Reply:
OK